### PR TITLE
fix object control so that zero efforts are applied for non-driven jo…

### PR DESCRIPTION
**Pull request by cremebrule**
…ints

### CODE REVIEW

File: omnigibson/objects/controllable_object.py [hunk 1]
```diff
@@ -436,8 +436,8 @@ def deploy_control(self, control, control_type, indices=None, normalized=False):
             elif ctrl_type == ControlType.POSITION:
                 joint.set_pos(ctrl, normalized=norm, drive=True)
             elif ctrl_type == ControlType.NONE:
-                # Do nothing
-                pass
+                # Set zero efforts
+                joint.set_effort(0, normalized=False)
```
Comment (ChengshuLi) [inline omnigibson/objects/controllable_object.py:440]: Could you remind me when ctrl_type will be `NONE` again?

File: omnigibson/prims/joint_prim.py [hunk 2]
```diff
@@ -866,6 +867,9 @@ def keep_still(self):
         Zero out all velocities for this prim
         """
         self.set_vel(np.zeros(self.n_dof))
+        # If not driven, set torque equal to zero as well
+        if not self.driven:
+            self.set_effort(np.zeros(self.n_dof))
```
Comment (ChengshuLi) [inline omnigibson/prims/joint_prim.py:872]: why not always set torque to 0?

File: omnigibson/utils/usd_utils.py [hunk 3]
```diff
@@ -124,7 +124,7 @@ def get_camera_params(viewport):
     view_proj_mat = helpers.get_view_proj_mat(view_params)
 
     return {
-        "pose": np.array(prim_tf),
+        "pose": np.array(prim_tf).T,
```
Comment (ChengshuLi) [inline omnigibson/utils/usd_utils.py:127]: hmmm why the transpose?

**Review by ChengshuLi — APPROVED** [review]
@cremebrule So sorry for the belated reply. Left some minor comments/questions. Feel free to merge.

File: omnigibson/prims/joint_prim.py (hunk 2 above)
Comment (cremebrule) [reply omnigibson/prims/joint_prim.py:872]: looping in @cgokmen . You mentioned we actually shouldn't set efforts directly now?

File: omnigibson/utils/usd_utils.py (hunk 3 above)
Comment (cremebrule) [reply omnigibson/utils/usd_utils.py:127]: I think empirically I just observed that the prim_tf is a transposed homogeneous matrix wrt to what we normally expect I think

**Pull request merged**